tests/secondTest/test2.1/tests.c

```diff
@@ -0,0 +1,76 @@
+#define _CRT_SECURE_NO_WARNINGS
+
+#pragma once
+#include "tests.h"
+#include "list.h"
+#include <stdio.h>
+
+bool emptyTest(void) {
+    FILE* file = fopen("emptyInput.txt", "r");
+    if (file == NULL) {
+        return false;
+    }
+    List* list = createList();
+    if (list == NULL) {
+        return false;
+    }
+    while (true) {
+        int number = 0;
+        if (fscanf(file, "%d", &number) != EOF) {
+            push(list, number);
+        }
+        else {
+            break;
+        }
+    }
+    fclose(file);
+    return isSymmetric(list) == true;
```

Comment:
```suggestion
    return isSymmetric(list);
```
Стайлгайд запрещает сравниваться с логическими константами